procrastinate/app.py

```diff
@@ -68,18 +67,7 @@ def __init__(
             this parameter.
             Raising this parameter can lower the rate of workers making queries to the
             database for requesting jobs.
-        job_store:
-            **Deprecated**: Old name of ``connector``.
         """
-        # Compatibility
-        if job_store:
-            message = (
-                "Use App(connector=procrastinate.PostgresConnector(...)) "
-                "instead of App(job_store=procrastinate.PostgresJobStore())"
-            )
-            logger.warning(f"Deprecation Warning: {message}")
-            warnings.warn(DeprecationWarning(message))
-            connector = job_store
         if not connector:
```

Comment:
Connector can now be a mandatory argument and this `if` branch can be removed.

Reply:
Yeah, I was too fast.

Reply:
Fixed.